packages/api/package.json

```diff
@@ -99,6 +99,7 @@
     "moment": "2.24.0",
     "morgan": "^1.9.1",
     "nodeify": "^1.0.1",
+    "node-forge": "^1.3.0",
```

Comment:
Why did this package get added here?

Reply:
It is just one of @Jkovarik's node-forge version related 4 commits (to master) that fixed my CI Audit, Lint failure. 
https://github.com/nasa/cumulus/pull/2896/commits/39bc7c55d59c9d6739989fe7a30d5ce4b83b5b8d

Reply:
Interesting. I think it that was a mistake in his changes, because `node-forge` isn't used in the API package. We did need to update it for the `common` package but shouldn't need it here. Let's try removing this and seeing if we get audit/lint failures.

Reply:
Removed node-forge from packages/api/package.json by reverting commit 39bc7c55d59c9d6739989fe7a30d5ce4b83b5b8d